docs/Overview.rst

```diff
@@ -36,17 +36,67 @@ MVS Validation Plan
 
-*Operational validity* determines if the model’s output is within the required accuracy. In order to achieve that, several validation techniques are used, namely:
+*Operational validity* assesses the model’s output with respect to the required accuracy. In order to achieve that, several validation techniques are used, namely:
 
-*	Benchmark testing, through which scenarios are created with different constraints and component combinations, and the output is calculated and compared to the expected one;
+* Graphical display, which is the use of model generated or own graphs for result interpretation. Graphs are simultaneously used with other validation techniques to inspect the results;
+
+*	Benchmark testing, through which scenarios are created with different constraints and component combinations, and the output is calculated and compared to the expected one to evaluate the performance of the model;
   
-*	Extreme scenarios (e.g., drastic meteorological conditions), created to make sure the simulation is through, then evaluate the output behavior by the use of graphs and qualitative analysis;
+*	Extreme scenarios (e.g., drastic meteorological conditions, very high costs, etc.), created to make sure the simulation is through and check if the output behavior is still valid by the use of graphs and qualitative analysis;
   
-*	Replication of one pilot project, with a validated optimization model with different component representation to compare and assess the results; and
+*	Comparison to other validated model, which replicates one pilot project into a validated optimization model with different component representation to identify the similarities and differences in results; and
   
 *	Sensitivity analysis, through which input-output transformations are studied to show the impact of changing the values of some input parameters.
 
+Implemented Tests
+#################
+
+Unit tests and integration tests are gauged by using test coverage measurement. The MVS covers so far 80% of the modules and sub-modules as seen in the next figure. Since those tests are automated, this coverage is updated for any changes in the model. The figure also shows that the build is currently passing.
+
+.. image:: images/Test_coverage.png
+ :width: 200
+
+A benchmark is a point of reference against which results are compared to assess the operational validity of a model. Benchmark tests are not automated like unit and integration tests, hence it is necessary to check that they are passing for any implemented changes in the model. The following table lists the implemented benchmark tests which cover several features and functionalities of the MVS.
```

Comment:
But benchmark tests are automated! The benchmark tests also always run though when pushing to master and should locally be run when pushing to dev.